### DISCUSSION TITLE
How do I update the About copy?

### DISCUSSION THREAD

**Discussion by popmechanic** [mediacms-io/mediacms#152] [Q&A]
I updated /templates/cms/about.html but the content did not update. Is this template file basically fixture data?

**Comment by Dan1ell** [comment]
I edited the paragraph inside block innercontent without any issue. 
What are you editing?

**Reply by popmechanic** [reply]
I edited the mediacms/templates/cms/about.html file.

**Reply by Dan1ell** [reply]
Yes, but which part of it? You cannot just replace the whole thing. You can edit between block innercontent and endblock.

**Reply by popmechanic** [reply]
I edited the p element inside of the innercontent block:

```
{% block innercontent %}
<div class="custom-page-wrapper">
        <h2>About</h2>
        <hr/>

    <p>[EDITED CONTENT]</p>
</div>
{% endblock %}
```

**Reply by Dan1ell** [reply]
Well, that should do it. Try this:
Navigate to your instance from a browser you haven't loaded the site with before. If your change is visible there, then you just need to clear the cache in the first browser. Getting rid of cookies isn't going to do it; specifically clear the cache.

**Reply by popmechanic** [reply]
Weird, cleared my cache AND tried it in a new browser. Still don't see the new content. I haven't yet gotten my head around the templating system. Is there any chance that content is stored in the db?

**Reply by Dan1ell** [reply]
No, it's right there on the page. Someone else will have to weigh in. Not sure what the issue is. 

BTW, are you using the docker installation or the single server installation, and what OS is this on?
And what browsers did you try? And finally, is there anything written in the debug.log?
(Covering all the bases.)

**Reply by popmechanic** [reply]
Single server, Ubuntu 20.04.2.0 LTS

**Reply by Dan1ell** [reply]
And what browsers did you try? 
And is there anything written in your debug.log?

**Reply by popmechanic** [reply]
Tried it in Brave and Safari. Cleared cache in Brave and tried it in Safari for the first time after changing the file.

Debug.log is empty.

**Reply by Dan1ell** [reply]
Besides the templates, are your other changes being reflected? Changes made to your settings file for instance. If you change the portal name does that show up?

**Reply by popmechanic** [reply]
Yep, I was able to change the settings file during installation. Since then I swapped out the logo files but haven't otherwise touched the codebase. When you say changing the portal name, do you mean via https://middlecore.video/admin/sites/site/?

**Reply by Dan1ell** [reply]
No just change the PORTAL_NAME variable in the settings file. And see if that change is reflected.
As far as I know, the value in the admin site is only used in email templates.

**Reply by popmechanic** [reply]
Aha! You know what fixed it? Restarting MediaCMS. So do I need to restart every time I make a change to a template?

**Reply by Dan1ell** [reply]
I guess so. On the docker-compose installation only changes to python and js code need a restart. I guess for the single server installation you need to restart for all changes. Glad it's solved!

**Reply by popmechanic** [reply]
Appreciate the help.
